routes.js

```diff
@@ -13,6 +13,8 @@ const responseSchema = {
 }
 
 module.exports = async function (app, opts) {
+  if (opts.routes === false) return
```

Comment:
Is this working? the `routes: false`  option is not passed in https://github.com/mcollina/fastify-gql/blob/master/index.js#L49-L51.

In fact, it's probably better to add it there instead.

Reply:
ah ok. was just quick and dirty hehehe